### PR TITLE
Increase colored text contrast in dark theme

### LINKED ISSUE #657: Improve readability of minimal tags
Change text colour for minimal tags to improve contrast and readability.

**Light theme:**

**Current**

screen shot 2017-02-09 at 7 36 29 pm

**Proposal**

screen shot 2017-02-09 at 7 38 34 pm

Default: no change
Primary: blue2 > blue1
Success: green2 > green1
Warning: orange2 > orange1
Danger: red2 > red1

**Dark Theme:**

**Current**

screen shot 2017-02-09 at 7 40 03 pm

**Proposal**

screen shot 2017-02-09 at 7 41 48 pm

Default: no change
Primary: blue4 > blue5
Success: green4 > green5
Warning: orange4 > orange5
Danger: red4 > red5

### PR DESCRIPTION
**Pull request by llorca**
#### Fixes #657 

Will add needed contrast to tags

cc @pkwi

### CODE REVIEW

File: packages/core/src/common/_mixins.scss [hunk 1]
```diff
@@ -20,10 +20,10 @@ $pt-intent-text-colors: (
 ) !default;
```
Comment (cmslewis) [inline packages/core/src/common/_mixins.scss]: #657 also proposes to darken intent text colors in light theme. Do we need to do that here too?

Comment (llorca) [inline packages/core/src/common/_mixins.scss]: Nope, @pkwi and I looked into it. We actually only want to do this for the dark theme

Comment (cmslewis) [inline packages/core/src/common/_mixins.scss]: K

**Comment by pkwi** [comment]
This is good

**Pull request merged**